support-frontend/assets/pages/contributions-landing/components/ContributionAmount.jsx

```diff
@@ -163,28 +167,86 @@ function withProps(props: PropTypes) {
     formatAmount(currencies[props.currency], spokenCurrencies[props.currency], { value: max.toString() }, false);
   const otherAmount = props.otherAmounts[props.contributionType].amount;
 
-  return (
-    <fieldset className={classNameWithModifiers('form__radio-group', ['pills', 'contribution-amount'])}>
-      <legend className={classNameWithModifiers('form__legend', ['radio-group'])}>How much would you like to give?</legend>
-      <ul className="form__radio-group-list">
-        {validAmounts.map(renderAmount(
+  const renderContribTypeChoiceCards = (shape: string) => {
+    const sharedCss = {
```

Comment:
@tjsilver can you put in a bit of documentation to explain why we are doing this so that whoever takes this test down will know what's happening here